pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -1,9 +1,18 @@
 # Arithmetic tests for DataFrame/Series/Index/Array classes that should
 # behave identically.
+from contextlib import (
+    AbstractContextManager,
+    nullcontext,
+)
 from datetime import (
     datetime,
     timedelta,
 )
+from functools import partial
+from typing import (
+    Type,
+    Union,
```

Comment:
We can't use `type` and `|`?

Reply:
I wish, that's [a `3.10` feature](https://docs.python.org/3/whatsnew/3.10.html#pep-604-new-type-union-operator).

Reply:
even if we do `from __future__ import annotations`?  im sure we use pipes elsewhere

Reply:
Huh, I'd tried that and switching to pipes, and it was failing for type aliases like `TD64_BOX_TYPE = TimedeltaArray | TimedeltaIndex | Series | DataFrame`. But TIL: pipes will work for function annotations (with that `import`).

`type` does work. I'll switch to that, and replace the aliases with pipes.

Reply:
Huh, I'd tried that and switching to pipes, and it was failing for type aliases like `TD64_BOX_TYPE = TimedeltaArray | TimedeltaIndex | Series | DataFrame`. But TIL: pipes will work for function annotations (with that `import`).

`type` does work. I'll switch to that, and replace the aliases with pipes.